infra/blueprint-test/modules/tft/terraform.go

```diff
@@ -0,0 +1,248 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package tft provides a set of helpers to test Terraform modules/blueprints.
+package tft
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	gotest "testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/modules/bpt"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/modules/discovery"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/modules/utils"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/mitchellh/go-testing-interface"
+	"github.com/stretchr/testify/assert"
+)
+
+// TFBlueprintTest implements bpt.Blueprint and stores information associated with a Terraform blueprint test.
+type TFBlueprintTest struct {
+	name        string                 // descriptive name for the test
+	fixtureName string                 // fixture name which will be used to compute TFDir
```

Comment:
If fixtureName is only used to compute tfDir do we need to store both?

Reply:
good point, removed